Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:3
+
+WORKDIR /wikirate
+
+RUN apt-get update && \
+    apt-get install -y imagemagick
+
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - && \
+    apt-get install -y nodejs
+
+COPY . .
+
+ENV RAILS_ENV=production
+
+RUN cp -R config/sample/* config && \
+    rm -R config/sample
+
+RUN bundle config without test cucumber cypress development profile
```

Comment:
alternatively you could copy a .bundle/config file. Not a dramatic difference, but maybe some day we'll want to tweak other bundle configurations...